CHANGES.rst

```diff
@@ -7,6 +7,18 @@ charge_migration
 - Added tests to see if the data array is changed after runnung the step and
   setting signal_threshold to 25000. [#7895]
 
+extract_1d
+----------
+
+- Move MIRI MRS residual fringe correction later in the code, and apply to surface
```

Comment:
Perhaps include a reason *why* it's getting moved, i.e. what effect does this change have?

Reply:
Added a more detailed note.  In brief, the residual fringe was getting applied in the stage where the spectrum is based on the aperture sum results; given that the area going into this sum can change for each wavelength plane the background or surf brightness spectra aren't necessarily robust until later when the normalization by the pixel area in the aperture has been done.